doc/Example-Gradle-Project/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,13 @@
+# This file is part of JavaSMT,
+# an API wrapper for a collection of SMT solvers:
+# https://github.com/sosy-lab/java-smt
+#
+# SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
```

Comment:
I updated and built the examples with 7.3.3 now. As this version complains about some things i did because they will become deprecated in the future i think this should be fine now? (I obviously updated everything that was complained about)
+ we removed all wrapper files anyway ;D